src/main/scala/viper/gobra/frontend/info/implementation/TypeInfoImpl.scala

```diff
@@ -112,6 +127,24 @@ class TypeInfoImpl(final val tree: Info.GoTree, final val context: Info.Context,
     res
   }
 
+  // TODO: computes every node that might contain a visible invocation to the importing package
+  private def relevantSubnodes(n: PNode): Vector[PNode] = n match {
+    // predicates, methods, pure methods
+    case decl@ PFunctionDecl(id, args, result, spec, _) =>
+      if (decl.spec.isPure) tree.child(decl) else id +: result +: spec +: args
```

Comment:
Isn't this equivalent to `tree.child(decl)` because the parser will throw away the body in case of an imported non-pure function & method?

Reply:
yes, you are right!